android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -79,6 +82,7 @@ public class RNSentryModuleImpl {
     private final PackageInfo packageInfo;
     private FrameMetricsAggregator frameMetricsAggregator = null;
     private boolean androidXAvailable;
+    private boolean debugEnabled = false;
```

Comment:
You can just check that directly from `HubAdapter().options.isDebug` or something similar.